saleor/settings.py

```diff
@@ -48,12 +48,13 @@ def get_bool_from_env(name, default_value):
 
 DATABASES = {
     "default": dj_database_url.config(
-        default="postgres://saleor:saleor@localhost:5432/saleor", conn_max_age=600
+        default=os.environ.get("DATABASE_STRING", "postgres://saleor:saleor@localhost:5432/saleor"), 
+        conn_max_age=600
     )
 }
 
 
-TIME_ZONE = "America/Chicago"
+TIME_ZONE = os.environ.get("TIME_ZONE", "America/Chicago")
```

Comment:
What is your use case for setting `TIME_ZONE` through env? You have different time zones in different instances/environments?

Reply:
@maarcingebala my TIME_ZONE is "Europe/Paris"
The goal is to left untouched settings.py between git pull.

Could you explain how to override this DATABASE_URL please ?

Reply:
It's an environment variable, you would use it exactly like the `DATABASE_STRING` that you proposed. But I think the solution that you really need is local settings file that is not tracked in Git.

In your repo, you should create a `local_settings.py` next to `settings.py`. Inside you need to import all your standard settings:

```python
# local_settings.py

from .settings import *

# here you can override any Django settings e.g.:

TIME_ZONE = "Europe/Paris"
```

When you have it, run your project with `DJANGO_SETTINGS_MODULE` pointing to that settings file e.g. `DJANGO_SETTINGS_MODULE=saleor.local_settings`. And that's it - when you run the project it should use your local settings file which is not tracked by Git and you can do there whatever you need for local configuration.

Reply:
@maarcingebala thanks for this tips :-)